website/src/en/playground.html

```diff
@@ -24,7 +24,7 @@
   <link rel="stylesheet" crossorigin  href="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.26.0/theme/neat.css">
 
   <script src="lib/prettier-version.js"></script>
-  <script src="playground.js"></script>
+  <script src="lib/playground.js"></script>
```

Comment:
Since you are moving it, we need to redirect users to the new playground url if they land on master. Here's the code I used in the past to do that:

```js
window.location = (window.location + '')
  .replace(/^[^#]+/, 'https://jlongster.github.io/prettier');
```

Note that we need to only do that if there's a hash and a okay heuristic would be if the hash starts with `{`